.github/workflows/k8s_psat_tpm_devid-pr-build.yaml

```diff
@@ -1,7 +1,14 @@
 name: PR Build
 
 on:
-  pull_request: {}
+  pull_request:
+    paths:
+    - 'k8s_psat_tpm_devid/**'
+
```

Comment:
To trigger jobs for a new plugin it will only be require to add the new path here?

Reply:
Right, if a new plugin wants to use GitHub Actions it will need to add its own workflow files with the `paths` property set to the directory where the plugin code lives.

Unfortunately GitHub Actions doesn't play well with multiple projects having different CI pipelines being in the same repository, so we need to:
- Make sure we trigger a workflow for a specific plugin only when we change code on its directory
- Set the working directory where the workflows will run